Ask your question UI improved #854

Sooraj002 opened the pull request:
## Issue that this pull request solves 

 Closes: #854 

## Proposed changes
Change of ui

### Brief description of what is fixed or changed
The elements were not centered , and close buttons position was wrong

## Types of changes
mainly css changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
![image](https://github.com/HITK-TECH-Community/Community-Website/assets/95868709/e3dd5a5c-60c9-4ceb-95b0-2ea305174c2c)
![image](https://github.com/HITK-TECH-Community/Community-Website/assets/95868709/5535292d-3fd4-4b5d-b458-7fb305e834e5)
![image](https://github.com/HITK-TECH-Community/Community-Website/assets/95868709/2eb23de5-d3dc-4a74-89c5-ccc5a0d6d0d5)

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

Kajol-Kumari reviewed (changes requested):
<img width="1493" alt="Screenshot 2024-05-11 at 9 58 16 PM" src="https://github.com/HITK-TECH-Community/Community-Website/assets/44888949/dcb5a22b-70ed-4e7a-8028-326526253126">

Seems like the responsiveness of the form is gone. Few things I am looking forward in this pr is - 
1. The `Tags` text should be aligned with its options below
2. Some padding around the close button
3. The look of the close button should be better.

Sooraj002 commented:
![image](https://github.com/HITK-TECH-Community/Community-Website/assets/95868709/657492a1-83a7-4047-80ab-785b17f6eb92)
I have added the things u have asked for but I deleted the repo i forked and now it is showing it is closed , So should i reopen the issue and then submit or can i directly add the pull request?
@Kajol-Kumari

Kajol-Kumari commented:
@Sooraj002 ofcourse the fix hasn’t been merged so you should either reopen this pr or open a new one
